src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.blok {
+  margin: auto;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.stars__star {
+  background-position: left;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)